src/components/toast/global_toast_list.test.tsx

```diff
@@ -7,15 +7,16 @@
  */
 
 import React from 'react';
-import { act } from '@testing-library/react';
+import { act, fireEvent } from '@testing-library/react';
 import { mount } from 'enzyme';
 import { requiredProps, findTestSubject } from '../../test';
-import { render } from '../../test/rtl';
+import { render, findByTestSubject, queryByTestSubject } from '../../test/rtl';
```

Comment:
These RTL APIs can be destructured directly from `render()` as opposed to being top level exports, which I believe is a general RTL best practice. See below comments for examples
```suggestion
import { render } from '../../test/rtl';
```

Reply:
As far as I know the [recommended RTL best practice for querying](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) is to use screen, I see now that we expose a custom screen util that has our custom matchers registered on it so I've opted to use it instead, let me know what you think

Reply:
Whoa, you're totally right - [TIL that RTL recommends screen](https://testing-library.com/docs/queries/about/#using-queries) for querying. Huh. Interesting!

To be totally honest with you, I don't always agree with Kent C Dodds (although he's obviously a super smart person) and some of the assertions he makes are very opinionated within very limited vs general contexts 😅 

Here's the reasons why I'd prefer to continue destructuring our queries from `render()` as opposed to importing them:

1. The rest of the EUI codebase and existing RTL tests do this. This is honestly probably the biggest reason; code consistency trumps almost all other factors.
2. There's very little impact on actual behavior (unless you need to query something outside the returned component container, e.g. portals), so basically we're just discussing syntactical sugar here. Just in my personal opinion, for 90% of use cases, using the returned queries is nicer because:
    1. It's fewer characters to type - e.g. `getByTestSubject('foo')` vs. `screen.getByTestSubject('foo')` vs. `getByTestSubject(container, 'foo')`
    2. It's encapsulated to the test only (so one test could use `getX` and the other could use `queryY` and it's very clear which test is using what)

For the purposes of this PR, I'd prefer to stick with the API/syntax we've been using across EUI for consistency, as mentioned, but I can certainly bring it to the rest of the EUI team after this to ask them whether they'd prefer to switch to `screen` to follow RTL's recommendations. But that's a change we would want to make across the codebase / all existing RTL tests to maintain consistency, and would be outside the scope of the current PR.

Hope that all makes sense - please let me know if not!

Reply:
Just to close the loop on this - the EUI team discussed this briefly in our sync today and agreed we generally prefer the `render()` destructured syntax. We're updating our contributing wiki (#7199) to note that this is a conscious deviation from RTL's recommendations.